pkg/util/util.go

```diff
@@ -30,3 +36,34 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+func FilterKubevirtLabels(labels map[string]string) map[string]string {
```

Comment:
This seems to be only relevant for the webhooks. Best to leave it close to it. Also, you already go over the labels. Can't you just apply your filter check here right now and only return labels which we care about?

Reply:
@rmohr because of it's usage in VMI update admitter as well, FilterKubevirtLabels should return all the kubevirt.io labels. But I've added a boolean to indicate whether we have restricted labels among all the kubevirt.io related.

Reply:
I don't think that it makes sense to forbid only a subset of labels on object creation but later on update block updating labels which were not blocked in the first place. What do you think?